src/types.ts

```diff
@@ -45,6 +45,11 @@ export type HandlerFunction<
   TTransformed
 > = (event: EmitterWebhookEvent<TName> & TTransformed) => any;
 
+export type RemoveHandlerFunction<
```

Comment:
Through experimentation, I found that the changes to `HandlerFunction` here: https://github.com/octokit/webhooks.js/pull/623/files#diff-c54113cf61ec99691748a3890bfbeb00e10efb3f0a76f03a0fd9ec49072e410aR43 were causing the blowup in duration.

I made `removeListener` have it's own variation `RemoveHandlerFunction`